modules/tile-converter/src/deps-installer/deps-installer.ts

```diff
@@ -87,14 +87,21 @@ export class DepsInstaller {
     console.log('Installing "join-images" npm package');
     const childProcess = new ChildProcessProxy();
     const nodeDir = dirname(process.execPath);
+    let spaceEscape = '\\ ';
+    let npmBinaryName = 'npm';
+    if (process.platform === 'win32') {
+      spaceEscape = '^ ';
+      npmBinaryName = 'npm.cmd';
+    }
     await childProcess.start({
-      command: `${nodeDir}/${process.platform === 'win32' ? 'npm.cmd' : 'npm'}`,
+      command: `${nodeDir.replaceAll(' ', spaceEscape)}/${npmBinaryName}`,
```

Comment:
I wonder if quotes work to support spaces in the path. It should work both on Windows and Linux:

```suggestion
      command: `"${nodeDir}/${npmBinaryName}"`,
```